changelog/unreleased/issue-3382

```diff
@@ -0,0 +1,8 @@
+Bugfix: honor RESTIC_CACHE_DIR environment variable by default
+
+--cache-dir option doesn't honor the RESTIC_CACHE_DIR environment variable by
+default. This fix an issue with the restic check command that doesn't obey the
```

Comment:
s/fix/fixes/

Mentioning the `check` command here is far too late. As it's the only affected command, it should already be mentioned in the changelog summary.